src/index.js

```diff
@@ -29,15 +29,29 @@ if (
 }
 
 // Event JSON example: https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#webhook-payload-example-32
-const eventJSON = JSON.parse(
-  readFileSync(process.env.GITHUB_EVENT_PATH, "utf8")
-);
+let eventJSON = JSON.parse(readFileSync(process.env.GITHUB_EVENT_PATH, "utf8"));
 
 // Examples of event types: "workflow_dispatch", "push", "pull_request", etc
 const eventType = process.env.GITHUB_EVENT_NAME;
 
 // Determine if the github user is an individual or an organization
-const githubUsername = eventJSON.repository.owner.login;
+let githubUsername = "";
+
+// eventJSON.repository is undefined for scheduled events
+if (eventType == "schedule") {
+  githubUsername = process.env.GITHUB_REPOSITORY_OWNER;
```

Comment:
Does this mean the user would have to set GITHUB_REPOSITORY_OWNER and GITHUB_REPOSITORY in their yaml file? Or are those env variables automatically set by GitHub?

If the user has to set them in the yaml, I wonder if there is some other way that could avoid that. For example, it looks like we might be able to do something like this:
```
const github = require('@actions/github');
...
githubUsername = github.repository_owner;
repoName = github.repository;
```

Sources:
* https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action#writing-the-action-code
* https://github.com/dhimmel/dump-actions-context
* https://www.npmjs.com/package/@actions/github

If nothing like that works and they simply have to set those as env variables, then we'll have to add instructions/examples to the readme.

Reply:
@weineran 
> Does this mean the user would have to set GITHUB_REPOSITORY_OWNER and GITHUB_REPOSITORY in their yaml file? Or are those env variables automatically set by GitHub?

The env vars are automatically set by GitHub. Source: https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables

> The default environment variables that GitHub sets are available to every step in a workflow.